### ISSUE TITLE
How is the external validation conducted?

### ISSUE THREAD

**Issue by ShugangZhang** [christianbosselmann/SCION#2]
Hi team, thanks for your impressive work. I am trying to use your model as a baseline, but I am confused about the external validation process in your paper. Specifically, there should be five models after the internal validation phase, which would give five predictions on the external validation set. However, only 1 result without any standard deviation is reported in Table 2. Is it a mean value?

Another possible explanation is that you have re-trained your model (after determining all its hyperparameters) based on a new division of the internal set (375 samples), and then tested the model on the external one. If this is the case, could you tell me how did you split this internal set?

I am asking because a different strategy in splitting the dataset may cause unfairness when comparing ours to your model. Thanks for your help. Looking forward to hearing from you.

**Comment by christianbosselmann** [comment]
Hi Shugang,

Thank you for your interest in SCION. For external validation, the model was first trained on the entire training data (n = 375) using the set of hyperparameters determined by k-fold cv. This model was then used to obtain predictions for an independent set of variants that were published after the last variant used in the training data (Table S1). Importantly, we ensured that this independent set did not contain any variants that were also present in the training data. 

Note that external validation / benchmarking is an ongoing issue in this field. Electrophysiological data is sparse, as it is expensive and time-consuming to generate. This 'external validation' set is therefore rather small, which is an important limitation when comparing models. If you want to be more rigorous, you could look into expanding this validation set with other published variants not used in training.

We are currently working on making SCION predictions available via command-line or batch upload. Meanwhile, predictions would need to be obtained via the shiny app (which can be somewhat tedious).

Hope this helps,

Christian

**Comment by ShugangZhang** [comment]
Thanks very much for your quick reply. Could you give more details on "the model was first trained on the entire training data (n = 375)"? For example, how do you stop training in this stage? I think it is not reasonable to use external test set as supervision, but if the test set is not used, how do you supervise the trainning in this stage? 

Maybe you have conducted a new splitting for the entire training set to separate a new validation set?

**Comment by christianbosselmann** [comment]
Again, the external validation set is at no point used for model training. Model training is done with k-fold cv on the training set. The Methods contain all neccessary information.
